persim/pers_landscape_approx.py

```diff
@@ -0,0 +1,380 @@
+"""
+    Persistence Landscape Approximate class
+
+    authors: Gabrielle Angeloro, Michael Catanzaro
+"""
+
+import numpy as np
+from operator import itemgetter, attrgetter
+from .pers_landscape import PersLandscape
+from .pers_landscape_aux import union_vals, ndsnap_regular
+
+__all__ = ["PersLandscapeApprox", "snap_PL", "lc_approx", "average_approx"]
+
+
+class PersLandscapeApprox(PersLandscape):
+    """
+    Persistence Landscape Approximate class.
+
+    This class implements an approximate version of Persistence Landscape,
+    given by sampling the landscape functions on a grid. This version is only
+    an approximation to the true landscape, but given a fine enough grid, this
+    should suffice for most applications. If an exact calculation with no
+    approximation is desired, consider `PersLandscapeExact`.
+
+    The default parameters for start and stop favor dgms over values. That
+    is, if both dgms and values are passed but start and stop are not, the
+    start and stop values will be determined by dgms.
+
+    Parameters
+    ----------
+    start : float, optional
+        The start parameter of the approximating grid.
+
+    stop : float, optional
+        The stop parameter of the approximating grid.
+
+    num_steps : int, optional
+        The number of dimensions of the approximation, equivalently the
+        number of steps in the grid.
+
+    dgms : list[list]
+        A list lists of birth-death pairs for each homological degree.
+
+    hom_deg : int
+        represents the homology degree of the persistence diagram.
+
+    vales
+
+    Methods
+    -------
+
+
+    Examples
+    --------
+
```

Comment:
I would love to see some examples and notebooks showing these new implementations off!

Reply:
For sure! We should be adding lots of examples in the next couple days.